### DISCUSSION TITLE
Nix

### DISCUSSION THREAD

**Discussion by aquifolly** [catppuccin/catppuccin#2325] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

creating a flake with the colours available as an output

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by isabelroses** [comment]
Is this not just https://github.com/catppuccin/catppuccin/issues/2103?

**Reply by aquifolly** [reply]
ah.

**Reply by aquifolly** [reply]
actually, no because it doesn't expose the colours directly so they can be used in applications without a port

**Reply by isabelroses** [reply]
might be worth going there and submitting a PR for that then, or a issue.
